insights/parsers/nmcli.py

```diff
@@ -191,3 +192,39 @@ def parse_content(self, content):
     def disconnected_connection(self):
         """(list): It will return all the disconnected static route connections."""
         return self._disconnected_connection
+
+
+add_filter(Specs.nmcli_conn_show_uuids, ["connection.uuid", "connection.id"])
+
+
+@parser(Specs.nmcli_conn_show_uuids)
+class NmcliConnShowUuids(CommandParser, dict):
+    """
+    This file will parse the output of nmcli connection detail information.
+
+    Sample configuration from an interface in file ``/usr/bin/nmcli conn show XXX``::
+
+        connection.id:                          System eth0
+        connection.uuid:                        5fb06bd0-0bb0-7ffb-45f1-d6edd65f3e03
+        connection.stable-id:                   --
+        connection.type:                        802-3-ethernet
+        connection.interface-name:              eth0
+        connection.autoconnect:                 yes
+
+    Examples:
+        >>> type(conn_show_uuids)
+        <class 'insights.parsers.nmcli.NmcliConnShowUuids'>
+        >>> conn_show_uuids["connection.uuid"]
+        '5fb06bd0-0bb0-7ffb-45f1-d6edd65f3e03'
+
+    Attributes:
+        data (dict): parameter-value format
+
+    """
+    def parse_content(self, content):
+        if not content:
+            raise SkipComponent()
```

Comment:
The codecov [reports](https://app.codecov.io/gh/RedHatInsights/insights-core/pull/4273?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=RedHatInsights#558b2a153764e7a4292d6a414684bb39-R225) that these two lines are not fully covered . @wushiqinlou  Shall we add a test-case for this?

Reply:
updated